README.md

```diff
@@ -56,47 +56,47 @@ Comparing with building from source manually, you will benefit from using this s
 SSH to your VPS and clone the Github repository:
 
 ```bash
-git clone https://github.com/masternodes/vps.git && cd vps
+https://github.com/thebitradio/Masternodes-VPS.git && cd Masternodes-VPS
```

Comment:
Should be:
git clone https://github.com/thebitradio/Masternodes-VPS vps && cd vps